projects/assets-library/src/icons/icon-type.ts

```diff
@@ -118,5 +118,10 @@ export const enum IconType {
   Warning = 'svg:warning',
   VirtualMachine = 'svg:virtual-machine',
   Yaml = 'svg:yaml',
-  ZoomToFit = 'zoom_out_map'
+  ZoomToFit = 'zoom_out_map',
+  PageLoader = 'svg:logo',
```

Comment:
can we rename the svg to `page-loader` as well? We may have another logo icon, so just want them to be named different for clarity